docs/guided-remediation.md

```diff
@@ -311,6 +683,17 @@ If you wish to apply your current relock & relaxation changes, select the "Write
 >
 > The `--relock-cmd` flag can be used to change the executed install command.
 
+### Override dependency versions remediation
```

Comment:
nit: having "remediation" at the end of all of these section headers reads a bit awkwardly. 

How about either:

"Remediation strategy: Override dependency verisons", or just "Override dependency versions"?